src/dataFiles.ts

```diff
@@ -7,12 +7,16 @@ import {
 } from "./constants.js";
 import { getRuntimeConfig } from "./runtimeConfig.js";
 
-function shouldIgnorePathHeuristically(root: string, isDir: boolean) {
+function shouldIgnorePathHeuristically(
+	root: string,
+	isDir: boolean,
+	isWebHook?: boolean,
```

Comment:
The reason that this was applied explicitly to webhook only is because once ensemble is merged, it will aggregate the episodes in a tv library during a search. What this provides is a way for webhook calls (this will pretty much always be from @bakerboy448 import script I've been exploring recently) to provide season packs from usenet to be searched after importing.

This would otherwise be impossible to get seeding as a pack immediately after download without kicking off a search. Ensemble does not aggregate from webhook calls.

Now that risky multi-file is merged, this means that if we search for the right thing and create the searchee properly, we can provide users a way to download season packs from usenet and actually produce season pack torrents from this.

I discussed some notification adjustments with Sonarr team and once those adjustments are made by markus (there is open issues for this) the feature will be usable.

Reply:
I'm still not fully understanding. Can you clarify the use case here? What do the webhook calls we're enabling look like?

Reply:
`curl -X POST http://cross-seed:2468/api/webhook?apikey=abcdef --data-urlencode "path=/media/tv/The Acolyte/Season 01"`

Just something like that.

It should parse the title out of the path and use the season number in the search with the title/id

Reply:
I just realized I didn't have this branch up to date and stuff.

Reply:
This isnt needed urgently as we are waiting on sonarr, but there are some things i just considered that will require some adjustments.

Just FYI.

Reply:
Why wouldn't we want to let this apply to all data based searchees?

Reply:
We don't need to...

>The reason that this was applied explicitly to webhook only is because once ensemble is merged, it will aggregate the episodes in a tv library during a search. These folders would otherwise be excluded by IGNORED_FOLDER_REGEX.

Reply:
I guess with cache it doesn't really matter....I'll just remove the leaky abstraction shit and do it that way.